axelrod/tests/unit/test_tournament.py

```diff
@@ -106,10 +106,10 @@ def test_serial_play(self):
             {'cooperation': [], 'payoff': []})
         self.assertFalse(tournament._run_parallel_repetitions.called)
 
-    #@given(s=lists(sampled_from(axelrod.strategies),
-    # Removing this as Hypothesis seems to have found a py2 bug.
-    # This is a temporary fix. For some reason mind reader seems to fail a test.
-    @given(s=lists(sampled_from([s for s in axelrod.strategies if s not in axelrod.cheating_strategies]),
+    # The @given below will run tests if the examples on lines 123... are
```

Comment:
Run and pass. This is not really useful, just in case anyone plays with this and wants to get tests passing.